server/src/main/java/org/apache/cassandra/sidecar/db/SystemAuthDatabaseAccessor.java

```diff
@@ -66,9 +68,7 @@ public String findRoleFromIdentity(String identity)
      */
     public Map<String, String> findAllIdentityToRoles()
     {
-        ensureIdentityToRoleTableAccess();
```

Comment:
Why are we removing this check?

Reply:
Since this is related to re-preparing PR. resolving this